t/test.c

```diff
@@ -148,7 +148,7 @@ static int on_stream_open(quicly_stream_open_t *self, quicly_stream_t *stream)
 
 quicly_stream_open_t stream_open = {on_stream_open};
 
-static void test_vector(void)
+static __attribute__ ((unused)) void test_vector(void)
```

Comment:
I think we might prefer retaining this, because we should reenable this test at some time.